llvm/unittests/CAS/CMakeLists.txt

```diff
@@ -1,3 +1,19 @@
+set(ONDISK_CAS_TEST_SOURCES
+  OnDiskGraphDBTest.cpp
+  OnDiskDataAllocatorTest.cpp
+  OnDiskKeyValueDBTest.cpp
+  OnDiskTrieRawHashMapTest.cpp
+  ProgramTest.cpp
+  )
```

Comment:
I think most other places unconditionally include tests for conditional features and then ifdef out all the test code based on what's in config.h. Any reason not to do that here too?